modules/EnsEMBL/Web/ImageConfig.pm

```diff
@@ -333,7 +333,7 @@ sub update_from_input {
     my $diff = delete $params->{$self->config_type};
 
     # Reset regulation matrix tracks by default
-    $self->altered($self->reset_user_settings('matrix', $params));
+    $self->altered($self->reset_user_settings('reg_matrix', $params));
```

Comment:
@msmanoj is that changes related to the JIRA ticket mentioned: https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6001. 
Because this code change seems to be fixing something for regulation matrix doesnt look like anything to do with species alignment issue. Maybe i am confused.